Good-Bids/goodbids
# Fix Auctioneer URL and Infinite Loop Issue

# Summary

Fixes Auctioneer URL and Infinite Loop bug.

## Issues

* N/A

## Testing Instructions

1. Use Admin as Administrator Role
2. Check Auctioneer URL

## Screenshots

<img width="693" alt="Screenshot 2024-02-27 at 8 43 55 PM" src="https://github.com/Good-Bids/goodbids/assets/122309362/07a43764-e2f1-409d-af7b-d16e70132b48">
